pkg/users/account-details.js

```diff
@@ -98,16 +98,32 @@ function get_expire(name) {
 
 export function AccountDetails({ accounts, groups, current_user, user, shells }) {
     const [expiration, setExpiration] = useState(null);
+    const [lastlogpath, setLastlogPath] = useState(null);
+
     useEffect(() => {
-        get_expire(user).then(setExpiration);
+        cockpit.spawn(["test", "-e", "/var/run/utmp"], { err: "ignore" }).then(() => {
+            setLastlogPath("/var/run/utmp");
+        }).catch(() => {
+            cockpit.spawn(["test", "-e", "/var/lib/lastlog/lastlog2.db"], { err: "ignore" }).then(() => {
+                setLastlogPath("/var/lib/lastlog/lastlog2.db");
+            }).catch(() => {
+                setLastlogPath(null);
```

Comment:
These 4 added lines are not executed by any test.

Reply:
unfortunately i do not know any distributions other than opensuse which uses lastlog2